build.gradle

```diff
@@ -207,6 +207,11 @@ subprojects {
     testImplementation libs.assertj.core
   }
 
+  tasks.withType(Test).configureEach {
+    // Some tests verify the messages included in the stack traces
```

Comment:
I don't think it's clear what this comment is trying to say and how it relates to the JVM arg

Reply:
I changed it the following sentence: 
> JVM may stop outputting stack trace of the exception which often occurs.
> Disable the optimization because some tests depend on the stack trace.